ha-jobs-core/src/main/scala/de/kaufhof/hajobs/JobManager.scala

```diff
@@ -124,6 +127,19 @@ class JobManager(managedJobs: => Jobs,
     val job = managedJobs(jobType)
     implicit val timeout = Timeout(5 seconds)
     (executor ? JobExecutor.Execute(job, triggerId)).mapTo[JobStartStatus]
+      .recoverWith {
+        case NonFatal(e) =>
+          logger.error(s"Error starting Job {} triggerId {}, set JobStatus to Failed!", jobType, triggerId, e)
+          retry(3)(jobStatusRepo.save(
+            JobStatus(triggerId, jobType, UUID.randomUUID(), JobState.Failed, JobResult.Failed, DateTime.now(), Some(Json.toJson(e.getMessage)))))
+            .map(_ => Error(s"Error starting Job $jobType triggerId $triggerId, set JobStatus to Failed! Msg: ${e.getMessage}"))
+            .recover {
+              case _ =>
+                logger.error(s"Error while writing JobStatus for job {} triggerId {}", jobType, triggerId, e)
```

Comment:
Here you probably want to log the error from save...? e is the wrong one and gets ignored anyways.